middleware/launch.test.js

```diff
@@ -85,4 +85,33 @@ describe("launcher middleware", () => {
 
     expect(next).toHaveBeenCalledWith(expect.any(Error));
   });
+
+  it("should convert date values to ISO string dates when building claims", () => {
+    expect(
+      buildClaims([
```

Comment:
We are holding date in two different formats?

Reply:
It seems so. We could look at storing date values in ISO format, but then you need to convert back and forth on each query/mutation assuming that the author wants to see the date in dd/mm/yyyy format - we could confirm with Sarah). Converting at the point of launch isolates the change to a single place.